src/contexts/states/LogFileManagerStore.ts

```
@@ -0,0 +1,18 @@
+import * as Comlink from "comlink";
+import {create} from "zustand";
+
+import {WrappedLogFileManager} from "../../services/WrappedLogFileManager";
```

ESLint

[error] 'WrappedLogFileManager' is defined but never used.

(@typescript-eslint/no-unused-vars)